pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -14,6 +14,33 @@ import (
 // HyperConvergedName is the name of the HyperConverged resource that will be reconciled
 const HyperConvergedName = "kubevirt-hyperconverged"
 
+// HyperConverged is the Schema for the hyperconvergeds API
+// +k8s:openapi-gen=true
+type HyperConvergedConfig struct {
+
+	// nodeSelector is the simplest recommended form of node selection constraint:
```

Comment:
the name in docs does no match the actual name of the attribute.

Reply:
done